code/processes/tickerlogreplay.q

```diff
@@ -188,15 +189,17 @@ replaylog:{[logfile]
  .replay.tablecounts:.replay.errorcounts:.replay.pathlist:()!();
 
  // If not running in segmented mode, reset replay date and clean HDB directory on each loop
+ .replay.zipped:$[logfile like "*.gz";1b;0b];
```

Comment:
if this is being overwritten here, is there any point in having the config value at top of file? Seems like this isn't really a config variable, more an internal flag to know what mode it's running in.

If so, I think line 31 should be removed, as it seems like configuring it differently won't make any difference.